pinc/graph_data.inc

```diff
@@ -857,3 +838,16 @@ function query_graph_cache($function, $args = [], $expiration = 3600)
 {
     return memoize_function($function, $args, $expiration, get_desired_language());
 }
+
+/**
+ * Given an array of numbers and a window, calculate the moving average over
+ * the window.
+ */
+function calculate_sma(array $array, int $period): array
```

Comment:
Given the function-comment, it would maybe make more sense to rename `$period` to `$window`.